packages/realm-server/tests/realm-server-test.ts

```diff
@@ -368,6 +368,34 @@ module('Realm Server', function (hooks) {
     assert.ok(response.headers['location'], '/person.gts');
   });
 
+  test('serves a card instance GET request with card-source accept header that results in redirect', async function (assert) {
+    let response = await request
+      .get('/person-1')
+      .set('Accept', 'application/vnd.card+source');
+
+    assert.strictEqual(response.status, 302, 'HTTP 302 status');
+    assert.strictEqual(
+      response.get('X-boxel-realm-url'),
+      testRealmURL.href,
+      'realm url header is correct',
+    );
+    assert.ok(response.headers['location'], '/person-1.json');
+  });
+
+  test('serves a card instance GET request with a .json extension and json accept header that results in redirect', async function (assert) {
+    let response = await request
+      .get('/person.json')
+      .set('Accept', 'application/vnd.card+json');
+
+    assert.strictEqual(response.status, 302, 'HTTP 302 status');
+    assert.strictEqual(
+      response.get('X-boxel-realm-url'),
+      testRealmURL.href,
+      'realm url header is correct',
+    );
+    assert.ok(response.headers['location'], '/person.json');
```

Comment:
I am confused with this one. It looks like it's redirecting from `/person.json` to `/person.json` (same url)?